deployment/pytorch/torch-hub-yolov5.ipynb

```diff
@@ -0,0 +1,905 @@
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "wZi8X-LoCORa"
+      },
+      "outputs": [],
+      "source": [
+        "endpoint = client.get_or_create_endpoint(NAME)"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "F4-9M2_xKF7R"
+      },
+      "outputs": [],
+      "source": [
+        "torch_cache_env = {'TORCH_HOME': '/tmp'}"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "id": "OHLC3QefHeFw",
+        "outputId": "c8af5b4d-621b-4cd3-f8bb-aa46a9d9b653"
+      },
+      "outputs": [],
+      "source": [
+        "%%time\n",
+        "endpoint.update(\n",
+        "    model,\n",
+        "    autoscaling = autoscaling,\n",
+        "    resources = resources,\n",
+        "    env_vars = torch_cache_env,\n",
+        "    wait = True\n",
+        ")"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "p7x7dfWoCNxK"
+      },
+      "outputs": [],
+      "source": [
+        "deployed_model = endpoint.get_deployed_model()"
+      ]
+    },
+    {
+      "attachments": {},
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "## 9. Predictions"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "4nS6rzzyK58c"
+      },
+      "outputs": [],
+      "source": [
+        "def process_image(url):\n",
+        "    return deployed_model.predict(url)"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "gH_5iPC84jmu"
+      },
+      "outputs": [],
+      "source": [
+        "urls = [\n",
+        "    ['http://www.cs.ucf.edu/~aroshan/index_files/Dataset_PitOrlManh/images/000001_0.jpg'],\n",
+        "    ['http://www.cs.ucf.edu/~aroshan/index_files/Dataset_PitOrlManh/images/000001_1.jpg'],\n",
+        "    ['http://www.cs.ucf.edu/~aroshan/index_files/Dataset_PitOrlManh/images/000001_2.jpg'],\n",
+        "    ['http://www.cs.ucf.edu/~aroshan/index_files/Dataset_PitOrlManh/images/000001_3.jpg'],\n",
+        "    ['http://www.cs.ucf.edu/~aroshan/index_files/Dataset_PitOrlManh/images/000001_4.jpg'],\n",
+        "    ['http://www.cs.ucf.edu/~aroshan/index_files/Dataset_PitOrlManh/images/000001_5.jpg']\n",
+        "]"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "XOKZr5qBJqup"
+      },
+      "outputs": [],
+      "source": [
+        "results = []\n",
+        "start_time = time.time()\n",
+        "\n",
+        "with concurrent.futures.ThreadPoolExecutor() as executor:\n",
+        "    for result in urls, executor.map(process_image, urls):\n",
```

Comment:
Small issue: I think this was meant to be

```suggestion
        "    for result in executor.map(process_image, urls):\n",
```

Otherwise, we get

```python
results[0] = urls
results[1] = executor.map(process_image, urls)
```

which doesn't seem right.

Reply:
Done.